UI/window-basic-adv-audio.cpp

```diff
@@ -237,7 +145,11 @@ void OBSBasicAdvAudio::SetShowInactive(bool show)
 		return;
 
 	showInactive = show;
-	activeOnly->setChecked(!showInactive);
+
+	ui->activeOnly->blockSignals(true);
+	ui->activeOnly->setChecked(!showInactive);
```

Comment:
Same here. We want to respect the Qt event flow. `on_activeOnly_toggled` is called when activeOnly has been toggled, so we shouldn't toggle it again..

Reply:
These have now been removed.